packer/nms/aws/nms.pkr.hcl

```diff
@@ -55,6 +55,11 @@ variable "subnet_id" {
   default = null
 }
 
+variable "nginx_installation_manager_version" {
```

Comment:
is this meant to be nginx_instance_manager?

Reply:
yes, it is. Updated.